### PR TITLE
[CI/SUBGRAPH] Workflow + Script for deploying to Satsuma endpoint

### LINKED ISSUE #1114: [SUBGRAPH] Add deployment workflow for Satsuma
Probably need to modify the CI file for subgraph deployments to play nicely with Satsuma.

### PR DESCRIPTION
**Pull request by 0xdavinchee**
- modify workflow for deploy-subgraph to support new deploy-to-satsuma.sh bash script

### CODE REVIEW

File: .github/workflows/handler.deploy-subgraph.yml [hunk 1]
```diff
@@ -58,8 +61,16 @@ jobs:
...
-      - name: "Deploy to endpoint"
+      - name: "Deploy to Superfluid endpoint"
+        if: inputs.version_label == ""
         run: "yarn deploy ${{ github.event.inputs.release_branch }} ${{ github.event.inputs.network }}"
         working-directory: ${{ env.subgraph-working-directory }}
         env:
           THEGRAPH_ACCESS_TOKEN: ${{ secrets.THEGRAPH_ACCESS_TOKEN }}
+
+      - name: "Deploy to Satsuma endpoint"
+        if: inputs.version_label != ""
```
Comment (kasparkallas) [inline .github/workflows/handler.deploy-subgraph.yml]: This is not straightforward logic... The environment the Subgraph is deployed to is chosen based on whether you passed in an optional "version label" argument? Please make it more explicit.

File: .github/workflows/handler.deploy-subgraph.yml [hunk 2]
```diff
@@ -12,6 +12,9 @@ on:
       network:
         required: false
         description: 'Network to deploy to (matic/xdai/kovan...)'
+      version_label:
```
Comment (kasparkallas) [inline .github/workflows/handler.deploy-subgraph.yml]: Maybe name it `satsuma_version_label` then?

File: .github/workflows/handler.deploy-subgraph.yml (hunk 1 above)
Comment (0xdavinchee) [reply .github/workflows/handler.deploy-subgraph.yml]: as you wish

File: .github/workflows/handler.deploy-subgraph.yml (hunk 2 above)
Comment (0xdavinchee) [reply .github/workflows/handler.deploy-subgraph.yml]: sure

**Comment by 0xdavinchee** [comment]
@kasparkallas pls re-review, requested changes have been applied

**Review by kasparkallas — APPROVED** [review]

**Review by hellwolf — APPROVED** [review]

**Pull request merged**